tool/jt.rb

```diff
@@ -653,8 +654,12 @@ def compile_cext(name, ext_dir, target, *clang_opts)
     Dir.chdir(ext_dir) do
       STDERR.puts "in #{ext_dir} ..."
       run('-rmkmf', "extconf.rb") # -rmkmf is required for C ext tests
-      raw_sh("make")
-      FileUtils::Verbose.cp("#{name}.su", target) if target
+      if File.exists?('Makefile')
```

Comment:
Added this to handle an extesion test that doesn't run on OSX: 
https://github.com/graalvm/truffleruby/blob/ec915bdcf1cccba94d60fd31c5f4b12d52c83919/test/mri/tests/cext/c/popen_deadlock/extconf.rb#L3